neuralmonkey/readers/plain_text_reader.py

```diff
@@ -53,7 +53,7 @@ def reader(files: List[str]) -> Iterable[List[str]]:
         text_reader = string_reader(encoding)
         for line in text_reader(files):
             io_line = io.StringIO(line.rstrip('\r\n'))
-            if quotechar is None:
+            if quotechar is not None:
```

Comment:
Neměla se tahle změna vyrebasovat pryč?

Reply:
tuhle změnu jsem udělal když jsem hrotil mypy. nechal bych to tady. takhle jak to je tak si myslim, že je to správně. @tomkocmi je to tak?

Reply:
prej takhle dobrý